popup.js

```diff
@@ -0,0 +1,109 @@
+const projects = [
+  {
+    project: {
+      title: 'Multi-post story',
+      src: 'images/appImage.png',
+      description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
+      technologies: ['html', 'Ruby on rails', 'css', 'Github'],
+      seeLiveLink: '#',
+      seeSourceLink: '#',
+    },
+  },
+];
```

Comment:
## WELL DONE 👏😁

- [ ] Your popup is working well and also responsive. Please kindly create objects for each project. Currently, you have just one object, if you fix this problem you should be approved by the next reviewer ✅. **_GOOD LUCK 🍀_**